.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "notifier-dev"]
+	path = notifier-dev
+	url = git@github.com:rsksmart/rif-marketplace-notifier.git
+[submodule "rns-dev"]
+	path = rns-dev
+	url = git@github.com:rsksmart/rif-marketplace-rns-dev-env.git
+[submodule "storage-dev"]
+	path = storage-dev
+	url = git@github.com:rsksmart/rif-marketplace-dev.git
```

Comment:
does it link itself?

Reply:
oh that's weird.... ok yea It seems I can't just use what's in the folder's package.json...